hooks/conf_regen/19-postfix

```diff
@@ -22,17 +22,19 @@ do_pre_regen() {
     main_domain=$(cat /etc/yunohost/current_host)
 
     # Support different strategy for security configurations
-    export compatibility="$(yunohost settings get 'security.postfix.compatibility')"
+    export compatibility="$(yunohost settings get 'security.postfix.postfix_compatibility')"
 
     # Add possibility to specify a relay
     # Could be useful with some isp with no 25 port open or more complex setup
     export relay_port=""
     export relay_user=""
-    export relay_host="$(yunohost settings get 'smtp.relay.host')"
-    if [ -n "${relay_host}" ]; then
-        relay_port="$(yunohost settings get 'smtp.relay.port')"
-        relay_user="$(yunohost settings get 'smtp.relay.user')"
-        relay_password="$(yunohost settings get 'smtp.relay.password')"
+    export relay_host=""
+    export relay_enabled="$(yunohost settings get 'email.smtp.smtp_relay_enabled')"
+    if [ "${relay_enabled}" == "True" ]; then
+        relay_host="$(yunohost settings get 'email.smtp.smtp_relay_host')"
+        relay_port="$(yunohost settings get 'email.smtp.smtp_relay_port')"
+        relay_user="$(yunohost settings get 'email.smtp.smtp_relay_user')"
+        relay_password="$(yunohost settings get 'email.smtp.smtp_relay_password')"
```

Comment:
Maybe if `yunohost settings get 'email.smtp'` works that could be a smarter way to fetch those settings with only one CLI call